docs/api/Agent.md

```diff
@@ -1,119 +1,95 @@
 # Agent
 
-## `new undici.Agent(opts)`
+Extends: `undici.Dispatcher`
```

Comment:
What's the difference between a Dispatcher and a Pool? Is Dispatcher some sort of an abstraction?

Reply:
Dispatcher is the base interface client, pool and agent implement.

Reply:
May contain pathname